controllers/front/CartController.php

```diff
@@ -653,15 +665,35 @@ private function checkCartProductsMinimalQuantities()
         foreach ($productList as $product) {
             if ($product['minimal_quantity'] > $product['cart_quantity']) {
                 // display minimal quantity warning error message
-                $this->errors[] = $this->trans(
-                    'The minimum purchase order quantity for the product %product% is %quantity%.',
-                    [
-                        '%product%' => $product['name'],
-                        '%quantity%' => $product['minimal_quantity'],
-                    ],
-                    'Shop.Notifications.Error'
+                $this->errors[] = $this->translateProductNameWithAttributes(
+                    'The minimum purchase order quantity for the product [1]%product%[/1] is [1]%quantity%[/1].',
+                    $product['name'],
+                    $product['attributes'],
+                    $product['minimal_quantity']
                 );
             }
         }
     }
+
+    /**
+     * @param string $sentence Sentence that contains "[1]%product%[/1]" (and "[1]%quantity%[/1]" in some)
+     * @param string $product_name The product name
+     * @param string|null $attributes The string of attributes
+     * @param int|null $quantity The quantity param (present in some sentences)
+     *
+     * @return string
+     */
+    private function translateProductNameWithAttributes(string $sentence, string $product_name, string $attributes = null, int $quantity = null): string
+    {
+        return $this->trans(
+            $sentence,
+            [
+                '%product%' => $product_name . (isset($attributes) && !empty($attributes) ? ' (' . $attributes . ')' : ''),
+                '%quantity%' => $quantity ?? '',
```

Comment:
:thinking: If `$quantity` not defined, won't `0` value be more explicit for  `The minimum purchase order quantity for the product [1]%product%[/1] is [1]%quantity%[/1].` ?